packages/eslint-config-airbnb/rules/style.js

```diff
@@ -85,7 +85,8 @@ module.exports = {
     // enforce padding within blocks
     'padded-blocks': [2, 'never'],
     // require quotes around object literal property names
-    'quote-props': 0,
+    // http://eslint.org/docs/rules/quote-props.html
+    'quote-props': [2, 'as-needed', { 'keywords': true, 'unnecessary': true, 'numbers': false }],
```

Comment:
Since we recommend using babel, which takes care of quoting reserved property names in legacy engines, this is not a concern.